run_gunicorn.py

```diff
@@ -30,7 +30,7 @@ def run_server():
     variables.
     """
     bind = f"0.0.0.0:{LISTEN_PORT}"
-    run(("gunicorn", "--log-config=logconfig.ini", "--log-level=debug", f"--bind={bind}", "run"))
+    run(("gunicorn", "--log-config=logconfig.yaml", "--log-level=debug", f"--bind={bind}", "run"))
```

Comment:
Does this actually work? Not only I am afraid that it doesn’t but we also found out that it’s this loading the configuration by the _--log-config_ argument that causes problems. Let’s remove it instead.

```suggestion
    run(("gunicorn", "--log-level=debug", f"--bind={bind}", "run"))
```

Reply:
Confirmed. It tries to use the _configparser_.

```
configparser.MissingSectionHeaderError: File contains no section headers.
file: 'logconfig.yaml', line: 1
'---\n'
```

Reply:
[This](https://github.com/benoitc/gunicorn/issues/1087#issuecomment-322502603) may be a solution. We already use _gunicorn.conf.py_ in production. I think we can move the logging configuration there! Not in this PR, but it’d definitely worth exploring.